packages/sdk/contractkit/src/wrappers/GrandaMento.ts

```diff
@@ -99,11 +120,34 @@ export class GrandaMentoWrapper extends BaseWrapper<GrandaMento> {
       approvalTimestamp: new BigNumber(result.approvalTimestamp),
       sellCelo: result.sellCelo,
       state,
+      id: exchangeProposalID,
+    }
+  }
+
+  async getHumanRedableExchangeProposal(
```

Comment:
wdyt of having the sell and buy amount as the decimal version? e.g. instead of
```
exchanger: 0x02779663A5719FB4f83abde16915e5556155745F
stableToken: Celo Dollar (cUSD) at 0x62b664dcD2a77ef60c5763EC433422158E783dFf
sellAmount: 6e+22
buyAmount: 6e+22
approvalTimestamp: 0
sellCelo: true
state: Proposed
id: 62
implictPricePerCelo: 1
```
It'd be:
```
exchanger: 0x02779663A5719FB4f83abde16915e5556155745F
stableToken: Celo Dollar (cUSD) at 0x62b664dcD2a77ef60c5763EC433422158E783dFf
sellAmount: 60000.0
buyAmount: 60000.0
approvalTimestamp: 0
sellCelo: true
state: Proposed
id: 62
implictPricePerCelo: 0.375
```

Just am thinking that it's easier to understand in terms of "whole" tokens rather than wei because the user will have to look up if the token is 18 decimals or what

Reply:
true, didn't consider that, sgtm